app/javascript/components/student/MentoringSession.tsx

```diff
@@ -85,6 +83,7 @@ export const MentoringSession = ({
         {discussion ? (
           <DiscussionInfo
             discussion={discussion}
+            mentor={mentor!}
```

Comment:
If we have a discussion, we should always have a `mentor`. You could instead do `discussion && mentor` on L83 if you prefered.

Reply:
Let's do `discussion && mentor`.